kafka-ui-react-app/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/enzyme": "^3.10.8",
```

Comment:
@Guzel738 Can you move newly added packages to `devDependecies`? So we can keep everything related to the tests in one place.